index.js

```diff
@@ -8,8 +8,13 @@ import {
   shouldBeBs58, verificationFail
 } from './helpers.js';
 import chai from 'chai';
+import {createRequire} from 'node:module';
 import {generateTestData} from './vc-generator/index.js';
-import {validVc} from './validVc.js';
+
+//FIXME remove this once node has non-experimental support
+//for es6 import and json
```

Comment:
```suggestion
```

That comment's going to be there for a very long time...and we use `createRequire()` throughout all these test suites. If someone cares to refactor it later, they can (and likely will). Just don't want to trip over a long lived todo list item.